api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -72,6 +101,7 @@ public boolean isRequesterEnabledInFirecloud() throws ApiException {
 
   @Override
   public Me getMe() throws ApiException {
+    getFirecloudStatus();
```

Comment:
So if I understand correctly, calling this will just return true or false but not do anything user-facing... I'm not sure you want to call this here?

Reply:
I don't actually want to do this here. Good catch, I'd forgotten I'd added that line...

Reply:
OK, please fix.